nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -417,6 +417,8 @@
 
 - The `shadowstack` hardening flag has been added, though disabled by default.
 
+- `xxd` is now provided by the `tinyxxd` package, rather than `vim.xxd`, to reduce closure size and vulnerability impact. Since it has the same options and semantics as Vim's `xxd` utility, there is no user impact. Vim's `xxd` remains available as the `vim.xxd` package.
```

Comment:
(I don’t want to commit to providing that separate output in Vim forever, but for at least a release it should be okay.)